.github/workflows/publish-runtime.yml

```diff
@@ -0,0 +1,166 @@
+name: Publish Runtime Draft
+
+# The code (like generate-release-body) will be taken from the tag versions, not master
+on:
+  workflow_dispatch:
+    inputs:
+      from:
+        description: tag (ex. runtime-53) to retrieve commit diff from
+        required: true
+      to:
+        description: tag (ex. runtime-155) to generate release note and srtool runtimes from
+        required: true
+
+jobs:
+  ####### Build runtimes with srtool #######
+
+  setup-scripts:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v3
+      - name: Upload scripts
+        uses: actions/upload-artifact@v3
+        with:
+          name: original-scripts
+          path: scripts
+      - name: Upload tools
+        uses: actions/upload-artifact@v3
+        with:
+          name: original-tools
+          path: tools
+
+  build-srtool-runtimes:
+    needs: ["setup-scripts"]
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        chain: ["dancebox"]
+        srtool_image:
+          - purestake/srtool
+        srtool_image_tag:
+          - 1.69.0
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.to }}
+      - name: Download original scripts
+        uses: actions/download-artifact@v3
+        with:
+          name: original-scripts
+          path: original-scripts
+      - name: Build & Push purestake/srtool image
+        if: github.repository == 'moondance-labs/tanssi'
+        run: |
+          docker pull "${{ matrix.srtool_image }}:${{ matrix.srtool_image_tag }}" && image_exists=true || image_exists=false
+
+          if [[ $image_exists = "false" ]]; then
```

Comment:
You might want to do something else than try to build it, as you won't be able to push to moonbeam docker hub from here.
I think simply making the script fail is enouigh